test/Snapshooter.Environment.Tests/Snapshooter.Environment.Tests.csproj

```diff
@@ -11,5 +11,20 @@
     <ProjectReference Include="..\..\src\Snapshooter.Xunit\Snapshooter.Xunit.csproj" />
     <ProjectReference Include="..\Snapshooter.Tests.Data\Snapshooter.Tests.Data.csproj" />
   </ItemGroup>
+  
```

Comment:
These dependencies are taken from /test/Testproject.props by default and do not need to be set in every project